ksqldb-engine/src/main/java/io/confluent/ksql/util/SharedKafkaStreamsRuntimeImpl.java

```diff
@@ -227,6 +227,14 @@ public synchronized void close() {
     kafkaStreams.cleanUp();
   }
 
+  @Override
+  public void pause(final QueryId queryId) {
```

Comment:
If this is a no-op in all implementations, does it even need to be part of the interface?

Reply:
It does not; removing it!

Likely, it was left-over from the original cribbing off of terminate query.